SportsNews/Controllers/AccountController.cs

```diff
@@ -153,5 +181,40 @@ private async Task<IdentityResult> CreateOrReplaceClaim(IdentityUser user, strin
             }
             return result;
         }
+
+        private async Task<string> ConvertFileToString(IFormFile file)
+        {
+            string base64String = String.Empty;
+            if (file != null)
+            {
+                using (MemoryStream m = new MemoryStream())
+                {
+                    await file.CopyToAsync(m);
+                    byte[] imageBytes = m.ToArray();
+
+                    base64String = Convert.ToBase64String(imageBytes);
+                }
+            }
+            return base64String;
+        }
+        private void SaveUserImage(string picture)
+        {
+            var userId = this.userManager.GetUserId(User);
+            var userPhoto = this.applicationDbContext.UserPhotos.FirstOrDefault(u => u.UserId == userId);
```

Comment:
This logic will be extracted within Repository Pattern. 
Extract this method to  UserRepository.cs  class and Create Method: 
SaveUserImage